node/runtime/src/lib.rs

```diff
@@ -58,8 +58,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("node"),
 	impl_name: create_runtime_str!("substrate-node"),
 	authoring_version: 10,
-	spec_version: 80,
-	impl_version: 82,
+	spec_version: 81,
+	impl_version: 81,
```

Comment:
Why's that decreased? I thought it is always-increasing value.

Reply:
spec_version increased.